docs/source/cli_help/status.rst

```diff
@@ -82,4 +82,26 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 
  .. note::
 
-    You can add additional space after ``└─`` and before branch name by setting git config key ``machete.status.extraSpaceBeforeBranchName``.
+    To make it easier to select branch name from the ``status's`` output on certain terminals (E.g. Alacritty), you can add additional
```

Comment:
Nit:
```suggestion
    To make it easier to select branch name from the ``status`` output on certain terminals (e.g. Alacritty), you can add additional
```
Plus maybe like to https://github.com/alacritty/alacritty (only here, not in docs.py)